src/components/skeleton/Header/Header.js

```diff
@@ -27,7 +27,9 @@ export default class Header extends Component {
   render() {
     var t = this.context.t;
     var loginOrProfile = !this.props.user.playerId ?
-      <FlatButton label={t('nav.login')} onClick={() => browserHistory.push(t.route('login'))} /> :
+      <div className="userLinks">
+      <FlatButton label={t('nav.login')} onClick={() => browserHistory.push(t.route('login'))} />
+      <FlatButton label={t('nav.changePassword')} onClick={() => browserHistory.push(t.route('changePassword'))} /> </div> :
```

Comment:
Hier toevoegen van links gaat niet omwille van mobile, dan zie je de "Erembodegem"-titel niet meer... :)

Ik zou het changePassword op de login pagina zelf doen.
--> ofwel daar een link naar de nieuwe pagina
--> ofwel naast de "LOGIN" knop, deze "PASWOORD VERGETEN" knop, dan is er geen nieuwe pagina nodig